components/tcf/ui/src/TCFContainer/TCFContainer.js

```diff
@@ -0,0 +1,66 @@
+import React, {Suspense, useState, useEffect} from 'react'
+import PropTypes from 'prop-types'
+
+const CONSENT_STATUS_NOT_ACCEPTED = 'NOT_ACCEPTED'
+const FirstLayer = React.lazy(() => import('@s-ui/sui-tcf-first-layer'))
+const SecondLayer = React.lazy(() => import('@s-ui/sui-tcf-second-layer'))
+
+export default function TCFContainer({
+  getVendorList,
+  getConsentStatus,
+  loadUserConsent,
+  saveUserConsent
+}) {
+  const [showFirstLayer, setShowFirstLayer] = useState(false)
+  const [showSecondLayer, setShowSecondLayer] = useState(false)
+
+  useEffect(() => {
+    async function checkConsentStatus() {
+      const consentStatus = await getConsentStatus()
+      const isConsentNotAccepted = consentStatus === CONSENT_STATUS_NOT_ACCEPTED
+      if (showFirstLayer !== isConsentNotAccepted) {
+        setShowFirstLayer(isConsentNotAccepted)
+      }
```

Comment:
you can do: setShowFirstLayer(consentStatus === CONSENT_STATUS_NOT_ACCEPTED)
If the state if the same, no rerender will be triggered